eslint.config.mjs

```diff
@@ -977,4 +979,143 @@ export default typescript.config([
       'no-restricted-imports': 'off',
     },
   },
+  {
+    name: 'plugin/boundaries',
+    plugins: {
+      boundaries,
+    },
+    settings: {
+      // order matters here in case of nested directories
+      'boundaries/elements': [
+        // --- specifics ---
+        {
+          type: 'devtoolbar',
+          pattern: 'sentry/components/devtoolbar',
+        },
+        {
+          type: 'core-button',
+          pattern: 'static/app/components/core/button',
+        },
+        {
+          type: 'core',
+          pattern: 'static/app/components/core',
+        },
```

Comment:
note: we can remove `core-button` once we’ve isolated everything in `core`. `core-button` is a separate group now so that we get fewer violations. Since we match with `core*`, both groups are included.